CHANGELOG.next.toml

```
@@ -79,3 +79,95 @@ message = "Fix cargo audit issue on criterion."
 references = ["smithy-rs#1923"]
 meta = { "breaking" = false, "tada" = false, "bug" = false }
 author = "ysaito1001"
+
+[[aws-sdk-rust]]
+message = """
+<details>
+<summary>The HTTP connector used when making requests is now configurable.</summary>
+
+```rust
+use std::time::Duration;
+use aws_smithy_client::{Client, hyper_ext};
+use aws_smithy_client::erase::DynConnector;
+use aws_smithy_client::http_connector::ConnectorSettings;
+use aws_types::SdkConfig;
+
+let https_connector = hyper_rustls::HttpsConnectorBuilder::new()
+    .with_webpki_roots()
+    .https_only()
+    .enable_http1()
+    .enable_http2()
+    .build();
+
+let smithy_connector = hyper_ext::Adapter::builder()
+    // Optionally set things like timeouts as well
+    .connector_settings(
+        ConnectorSettings::builder()
+            .connect_timeout(Duration::from_secs(5))
+            .build()
+    )
+    .build(https_connector);
+
+let sdk_config = SdkConfig::builder()
+    .http_connector(smithy_connector)
+    .build();
+
+let client = Client::new(&conf);
```

```suggestion
let client = Client::new(&sdk_config);
```
Be sure to compile/run this 😉